crates/moonlight-cli/src/main.rs

```diff
@@ -60,8 +60,7 @@ fn main() -> eyre::Result<()> {
             let install = detect_install(&exe);
             if let Some(install) = install {
                 if install.patched {
-                    log::warn!("Install already patched");
-                    std::process::exit(0);
+                    installer.unpatch_install(&install.install)?;
```

Comment:
We should probably warn the user we're unpatching the client - I know that the installer sees hh and maybe Vencord as "patched" even though it's a different mod.